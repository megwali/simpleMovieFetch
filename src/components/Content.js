import { Route, Switch } from 'react-router-dom';
import { Home, Search, Show } from '../pages';
import { Container } from './StyledComponents';

const Content = () => (
  <Container>
    <Switch>
      <Route exact path="/">
        <Home />
      </Route>

      <Route path="/search">
        <Search />
      </Route>

      <Route path="/show">
        <Show />
      </Route>
    </Switch>
  </Container>
);

export default Content;
