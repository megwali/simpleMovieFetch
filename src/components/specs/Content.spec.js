import { create } from 'react-test-renderer';
import Content from '../Content';
jest.mock('react-router-dom', () => ({ Link: 'Link', Route: 'Route', Switch: 'Switch' }));
jest.mock('../StyledComponents', () => ({ Container: 'Container' }));


describe('Content', () => {
  it('renders', () => {
    const component = create(<Content />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
