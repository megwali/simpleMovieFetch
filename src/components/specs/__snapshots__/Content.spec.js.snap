// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders 1`] = `
<Container>
  <Switch>
    <Route
      exact={true}
      path="/"
    >
      <div>
        <p>
          You don't have any shows in your watchlist
        </p>
      </div>
    </Route>
    <Route
      path="/search"
    >
      <div
        className="sc-eCssSg jLJXbU"
      >
        <input
          className="sc-jSgupP ctZjyU"
          onChange={[Function]}
          type="search"
        />
        <div
          className="sc-hKgILt gbuHto"
        />
      </div>
    </Route>
    <Route
      path="/show"
    >
      <div>
        <div
          className="sc-fubCfw bDJXPO"
        >
          <img
            className="sc-pFZIQ iPfVUh"
          />
          <div>
            <p />
            <h4 />
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": undefined,
                }
              }
            />
          </div>
        </div>
        <button
          onClick={[Function]}
          style={
            Object {
              "padding": "0.5rem",
            }
          }
        >
          Add to watchlist
        </button>
      </div>
    </Route>
  </Switch>
</Container>
`;
